recipes/openblas/bld.bat

```diff
@@ -0,0 +1,11 @@
+
+:: Set $HOME to the current dir so msys runs here
+set HOME=%cd%
+
+:: Configure, build, test, and install using `make`.
+bash -lc "make FC=gfortran DYNAMIC_ARCH=1 BINARY=$ARCH NO_LAPACK=0 NO_AFFINITY=1 USE_THREAD=1 PREFIX=`echo $LIBRARY_PREFIX | tr '\\' '/'`"
```

Comment:
Yeah, tried this before. That ends up being a bad idea.

The only reason one wants this is for threading, of course, which I am already leaning towards dropping entirely.

The problem is gomp (GNU OpenMP) is not fork safe. So, when someone uses `multiprocessing` and `numpy`, etc. (built with OpenBLAS using GNU OpenMP), they will get horrible crashes. If we used a different OpenMP implementation, this might not be a problem, but, of course, this would require a different compiler. As that switch is highly unlikely at this point, the smart move is to steer clear of OpenMP here. I believe @njsmith made a [patch]( https://gcc.gnu.org/bugzilla/show_bug.cgi?id=60035 ) for `gcc` that would address this awhile back, but I don't know whatever happened to it.

The safe way is to use pthreads (POSIX Threads) here, which is what we are doing, but that has had performance problems in the past.